src/components/index/sections/SupportUsSection.tsx

```diff
@@ -2,16 +2,20 @@ import { Grid, Typography, Box } from '@material-ui/core'
 import { useTranslation } from 'react-i18next'
 import { makeStyles, createStyles } from '@material-ui/core/styles'
 
-import FinancesIcon from '../../icons/support-us-icons/FinancesIcon'
-import LabourIcon from '../../icons/support-us-icons/LabourIcon'
-import MediaIcon from '../../icons/support-us-icons/MediaIcon'
-import PartnershipIcon from '../../icons/support-us-icons/PartnershipIcon'
+import FinancesIcon from '../icons/support-us-icons/FinancesIcon'
+import LabourIcon from '../icons/support-us-icons/LabourIcon'
+import MediaIcon from '../icons/support-us-icons/MediaIcon'
+import PartnershipIcon from '../icons/support-us-icons/PartnershipIcon'
 
 const useStyles = makeStyles((theme) =>
   createStyles({
     icon: {
       fontSize: theme.spacing(10),
       fill: 'none',
+      marginBottom: '7px',
```

Comment:
You can use `theme.spacing(1)` here to match the theme global spacing rate

Same for the margin bottom